web-app/src/main.ts

```diff
@@ -18,6 +18,7 @@ async function bootstrap() {
       saveUninitialized: true,
     }),
   );
+  app.enableCors();
```

Comment:
We could be more selective and allow only our local Expo web app to consume our API

Reply:
Why not? Its destiny is to become a public API one day, isn't it? 😁

Reply:
Yep, one day but not for now :p

Reply:
As seen together, this was finally done with PR https://github.com/marmelab/Hex/pull/46
For now I left this configuration (allow all origins) because I did not have enough time to test the config allowing only one origin.